docs/web/user_guide.md

```diff
@@ -616,9 +616,16 @@ filter arguments:
                         <minimum_bug_path_length>:<maximum_bug_path_length>.
                         Valid values are: "4:10", "4:", ":10"
   --tag [TAG [TAG ...]]
-                        Filter results by version tag names. This can be used
-                        only if basename or newname is a run name (on the
-                        remote server).
+                        Filter results by version tag names.
+  --open-reports-date TIMESTAMP
+                        Get results which were detected BEFORE the given date
+                        and NOT FIXED BEFORE the given date. The detection
+                        date of a report is the storage date when the report
+                        was stored to the server for the first time. The
+                        format of TIMESTAMP is
+                        'year:month:day:hour:minute:second' (the "time" part
+                        can be omitted, in which case midnight (00:00:00) is
```

Comment:
What do you think about setting the default value to 24:00:00?
If I select the day only it will not show the results on that day until if I set the time to the end of the day the current 0:00:00 is the beginning of that day.

Reply:
If we change this behavior do not we need to change it for other filters too (--detecter-before, --fixed-before)?

By the way you can always reach the same behavior if you don't specify the time but just increase the date with +1 day. For example instead of writing 2020:09:04 you can write 2020:09:05.

Reply:
yes, but I think this is a bit counter intuitive to set the date to the next day if I want to see the results for today. @bruntib @dkrupp @gamesh411 what do you think?

Reply:
It's not very intuitive at first glance, but at least it is consistent. The documentation clears it up: detected before and unfixed before. Is noon of today *`before`* today? Not really. I believe consistency should win this round.

In addition, this would be slippery slope, 'cuz what about only specifying a month, or a year? What if I give hours and minutes, but not seconds? If I say `11:40` then should `11:40:30` count, or not? I think not. You're specifying a left or right side demarcation line.

Maybe an error log entry could be added when the query is executed which explicitly explains to the user

    Querying reports fixed before yyyy-MM-dd HH:mm:ss....

In which the user will clearly see what timestamp their CLI argument was expanded to.

Reply:
yes a log entry can help but with a json format it would mess up the returned json, so there will not be any extra information